scripts/security/ldap_users/21_restAdmin.ldif

```diff
@@ -0,0 +1,15 @@
+dn: cn=restAdmin,ou=users,{{ LDAP_BASE_DN }}
```

Comment:
@sdanduConf for consistency, perhaps `restAdmin` user should also be added to the group ldif, similar to `connectAdmin` at https://github.com/confluentinc/cp-demo/blob/5.5.0-post/scripts/security/ldap_users/20_group_add.ldif#L26-L29

Reply:
Added, thanks!